v22.1/restore.md

```diff
@@ -788,7 +802,7 @@ RESTORE DATABASE bank FROM LATEST IN 'gs://{bucket name}?AUTH=specified&CREDENTI
 
 ### Restore a backup asynchronously
 
-Use the `DETACHED` [option](#options) to execute the restore [job](show-jobs.html) asynchronously:
+Use the [`DETACHED`](#detached) [option](#options) to execute the restore [job](show-jobs.html) asynchronously:
```

Comment:
Similar comment to above, these links are more or less going to the same place. (#detached and #options), can we pick one for the sentence?

Reply:
picked #detached